modules/javafx.controls/src/test/java/test/javafx/scene/control/MenuItemTest.java

```diff
@@ -419,22 +406,36 @@
         assertEquals(other.get(), menuItem.isVisible());
     }
 
-    @Ignore // keyCharacter for keyCodeCombination cannot be null
-    @Test public void setSpecifiedAccelerator1() {
+    @Test(expected=NullPointerException.class)
+    public void setSpecifiedAccelerator_nullKeyCombination1() {
```

Comment:
Constructing KeyCodeCombination with a null KeyCode & constructing a KeyCharacterCombination with null key character results in NullPointerException as present in KeyCombinationTest.java.

These tests were ignored with a comment on similar lines. I have enabled them and added expected NullPointerException.

Reply:
Where is the NPE thrown? I see that your change makes the test pass, but if it never gets to the `setAccelerator` call or the `assertEquals` statements maybe you could remove them?

Reply:
You are right. We can remove these test cases.
The NPE is thrown from KeyCodeCombination and KeyCharacterCombination constructors and not from the method of MenuItem that is under test.